GitCommands/Git/GitModule.cs

```diff
@@ -1778,13 +1811,13 @@ public string UnstageFiles(IReadOnlyList<GitItemStatus> files)
 
             if (nonNewFiles.Count != 0)
             {
-                StringBuilder sb = new("reset --");
+                GitArgumentBuilder sb = new("reset") { "--" };
                 foreach (var file in nonNewFiles)
                 {
-                    sb.Append($" \"{file.Name.ToPosixPath()}\"");
+                    sb.Add($"\"{file.Name.ToPosixPath()}\"");
                 }
 
```

Comment:
Quote or QuoteNe extension method?

Reply:
Are we sure removing the space still works as expected?

Reply:
Nevermind. I get it. The add adds the space.

Reply:
changed to QuoteNE